itou/api/applicants_api/views.py

```diff
@@ -52,3 +26,28 @@ def get_queryset(self):
             .prefetch_related("job_applications")
             .order_by("-pk")
         )
+
+
+ApplicantsView.__doc__ = f"""\
+# Liste des candidats par structure
+
+Cette API retourne la liste de tous les demandeurs d'emploi liés à une candidature pour la structure en cours.
+
+Les candidats sont triés par date de création dans la base des emplois de l'inclusion,
+du plus récent au plus ancien.
+
+# Permissions
+
+L'utilisation de cette API nécessite un jeton d'autorisation (`token`) :
+
+{AUTH_TOKEN_EXPLANATION_TEXT}
+
+Ce compte doit être **uniquement membre** de la structure dont on souhaite récupérer la liste de candidats.
```

Comment:
`AUTH_TOKEN_EXPLANATION_TEXT` indique qu'il faut récupérer le token en tant qu'administrateur (et non membre), ce qui peut porter à confusion (même si le sens n'est pas le même ici).

Peut-être:
```suggestion
Le compte administrateur utilisé ne doit être membre **que** de la structure dont on souhaite récupérer la liste de candidats (et non membre de plusieurs structures).
```